### ISSUE TITLE
When I add the chinese task will crash.

### ISSUE THREAD

**Issue by CG-man** [kdheepak/taskwarrior-tui#121]
**Describe the bug**

**To Reproduce**

**Environment (please complete the following information):**

- Operating System: 
Mac

- Installation: 
homebrew

- taskwarrior-tui version:

```bash
taskwarrior-tui --version
taskwarrior-tui 0.5.3
```
- task version:

```bash
task --version
2.5.3
```

**Additional context and information**

![image](https://user-images.githubusercontent.com/12809498/109111028-2758af80-7773-11eb-8aad-8bb511e5fcc7.png)

**Comment by kdheepak** [comment]
Thanks for opening an issue. If you update to a more recent version of `taskwarrior-tui` this issue should be resolved.

**Comment by CG-man** [comment]
> Thanks for opening an issue. If you update to a more recent version of `taskwarrior-tui` this issue should be resolved.

yep, the problem is resolved, 3Q, tt v0.5.3 is install use the cargo
``` shell
cargo install taskwarrior-tui
```

**Comment by kdheepak** [comment]
Yeah, the `cargo install taskwarrior-tui` is lagging because I hadn't tagged a dependency crate. I'll get to it eventually, things are a little in flux right now.
